js/huobi.js

```diff
@@ -2447,6 +2448,31 @@ module.exports = class huobi extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetMarginLoanInfo (params);
```

Comment:
We will need to use the new `spot` API endpoint here, `spotPrivateGetV1MarginLoanInfo`. The old endpoints are deprecated: 

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L109
- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L252